packages/static-wado-webserver/lib/index.mjs

```diff
@@ -92,6 +95,10 @@ const methods = {
     await addQueryCall(router, "/studies", params, "studyQuery");
     router.get("/studies", qidoMap);
     router.get("/studies/:studyUID/series", qidoMap);
+
+    // TODO we definitely should move all other routes def into this approach.
+    setRoutes(router);
```

Comment:
Can you call this setDicomwebRoutes?  There may well be other routes setup for other types of things in the future.

Reply:
setDicomwebRoutes would be something inner setRoutes. The last one is (or at least should be) the entry point of every route definition, instead having them all fragmented on here on this file.